org.jacoco.core/src/org/jacoco/core/instr/Instrumenter.java

```diff
@@ -114,7 +114,7 @@ protected String getCommonSuperClass(final String type1,
 				return instrument(new ClassReader(buffer));
 			}
 		} catch (final RuntimeException e) {
-			throw instrumentError(name, e);
+			throw instrumentError("class " + name, e);
```

Comment:
@marchof Here is a story: at first I also wanted to remove word "class", then realized that `Instrumenter` is used by `CoverageTransformer` and [`CoverageTransformerTest` expects word "class"](https://github.com/jacoco/jacoco/blob/30bc8e190e9a90fc43f5f68c499062cd58db3ebe/org.jacoco.agent.rt.test/src/org/jacoco/agent/rt/internal/CoverageTransformerTest.java#L215), where BTW `name` is not a name of file but a name of class and so without extension `.class`. After that instead of removal I pulled this word up from `instrumentError` into existing places of usage `instrumentError`, preserving existing error messages. And then added new usages of `instrumentError` without word "class" into places where `name` does not refer to a class.

Also "archive.jar@class something" is impossible - word "class" can't appear in the middle, because its addition happens at the very end when `name` is already fully constructed, and its addition happens only in places where we know that `name` refers to a class.

And so alternatively there are two other options - move addition of prefix "class" into `CoverageTransformer` or completely remove it. WDYT?